core/domain/exp_services.py

```diff
@@ -816,6 +818,23 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
 
     exploration.version += 1
 
+    if feconf.ENABLE_STRING_CLASSIFIER:
```

Comment:
Why is this constant specific to string classifier? We are going to have many more. Why not use more general constant like ENABLE_ML_CLASSIFIERS or ENABLE_MACHINE_LEARNING_CLASSIFIER or something similar?